### ISSUE TITLE
kr1-desktop: save game import missing stars

### ISSUE THREAD

**Issue by cigumo** [cigumo/krli#1289]
https://steamcommunity.com/app/246420/discussions/2/2139714324765906612/

Sharelink: https://we.tl/t-WNMKoyXlDJ
Game Folder contains the folder content of the LOVE Version
Slot1.zip contains the original SteamApps Save-file

> i had beaten anything. Now i am down to 114 Stars out of 130.

![](https://i.ibb.co/jzCrrB8/KR-Beta-Map.jpg)

**Comment by cigumo** [comment]
I've downloaded your original save games and imported it with the new version of the game.
These are my findings:

- The original save game, for some unknown reason, reports 114 of 130 stars.
- After opening this save game in the Beta version it properly reports 130/130 stars.
- Both the original and imported save game report the same achievements complete: all done except: Twin Rivers Angler, Toxicity, Are you not entertained?.
- The two new achievements show in the imported versions incomplete, as they should.

In summary, something happens in the original version of the game that shows less stars than the save game reports, but don't worry, as you will not loose any progress after we release the new version.
